internal/test/scorecard/common_utils.go

```diff
@@ -450,6 +455,66 @@ func updateAndWaitTillCryostatAvailable(cr *operatorv1beta1.Cryostat, resources
 	return err
 }
 
+func CheckRolloutStatus(name string, namespace string, resources *TestResources) error {
```

Comment:
Maybe we refactor this common rollout check? Because in `updateAndWaitTillCryostatAvailable` also uses the same logic.

Reply:
Though, I think we just need to call `waitForDeploymentAvailability` instead for the report deployment.